tree/ntuple/v7/inc/ROOT/RNTupleSerialize.hxx

```diff
@@ -69,7 +68,7 @@ public:
    static constexpr DescriptorId_t kZeroFieldId = std::uint64_t(-2);
 
    struct REnvelopeLink {
-      std::uint32_t fUnzippedSize = 0;
+      std::uint64_t fUnzippedSize = 0;
```

Comment:
Maybe this is an instance where the convention that `size` is for compressed size and `length` for uncompressed size is not respected